config/routes.js

```diff
@@ -104,4 +104,10 @@ module.exports.routes = {
   '/test/loadSeasonFixture': 'TestController.loadSeasonFixture',
   '/test/loadMatchFixtures': 'TestController.loadMatchFixtures',
   '/test/loadFinishedGameFixtures': 'TestController.loadFinishedGameFixtures',
+  '/test/game': 'TestController.getGames',
+  '/test/match': 'TestController.getMatches',
+
+  // Catch all for Vue paths only require get for http request so we don't interfer
+  // with any blueprints routes
+  '/*': { controller: 'ViewController', action: 'serveIndex', skipAssets: true },
```

Comment:
I considered importing the routes from vue router in to this file and making it only proxy vue routes, but then we lose the benefit of vue handling any non app route where we'd want a true Vue powered 404. Speaking of which, @itsalaidbacklife -- we'll need a todo for a proper 404 now once this goes in.

Maybe some sort of animated Cuttleworth, floating upside down with x eyes, hahahaha.